src/components/MoviesList/MoviesList.js

```diff
@@ -1,6 +1,23 @@
 import React from 'react';
+import PropTypes from 'prop-types';
+import { MovieCard } from '../MovieCard';
 import './MoviesList.scss';
 
-export const MoviesList = () => (
-  <>Put the list here</>
+export const MoviesList = ({ moviesFromServer }) => (
+  <div className="movies">
+    {moviesFromServer.map(movie => (
+      <MovieCard
+        key={movie.imdbId}
+        {...movie}
+      />
```

Comment:
```suggestion
      <MovieCard key={movie.imdbId} {...movie} />
```
2 пропса можем писать в 1 строку, как в html